src/Visitor/MongoOdm.php

```diff
@@ -73,6 +78,26 @@ public function visit(OperationInterface $operation)
     }
 
     /**
+     * Provides the  Doctrine QueryBuilder
+     *
+     * @return QueryBuilder
+     */
+    public function getBuilder()
+    {
+        return $this->queryBuilder;
+    }
+
+    /**
+     * Provides the Doctrine Query object to execute.
+     *
+     * @return \Doctrine\ODM\MongoDB\Query\Query
+     */
+    public function getQuery()
```

Comment:
Do we really need this method? It doesn't look like you're using it in libgraviton/graviton#68